### PR TITLE
i12 resolved - change request id calc error

### PR DESCRIPTION
**Pull request by RahulR100**
The function to increment the ID was broken when given numbers 0.008 and 0.009. This has been fixed by rounding the final result to 3 decimal places before converting to a string. Now 0.008 returns 0.009 instead of 0.0089999.... etc.

### CODE REVIEW

File: server-code/ChangeForm.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ function getNextChangeRequestId(requestType) {
         return typeLetter + "-001";
     } else {
         var numStr = "0." + latestId.substring(latestId.indexOf("-") + 1); // converts ID (N-001) into number string ("0.001")
-        var nextNum = (parseFloat(numStr) + 0.001).toString(); // converts string to float and increments by 1
+        var nextNum = (parseFloat(numStr) + 0.001).toFixed(3).toString() // converts string to float and increments by 1
```
Comment (jamescd18) [inline server-code/ChangeForm.js:39]:
```suggestion
        var nextNum = (parseFloat(numStr) + 0.001).toFixed(3).toString(); // converts string to float and increments by 1
```

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Quick fix. Also, JSDoc would be much appreciated on functions being edited. Thank you!

File: server-code/ChangeForm.js (hunk 1 above)
Comment (RahulR100) [reply server-code/ChangeForm.js:39]: Oop.. always those semicolons ahaha

File: server-code/ChangeForm.js [hunk 2]
```diff
@@ -3,8 +3,11 @@ Document: JS code connecting the Change Management Request form to the spreadshe
 
 */
 
-// onRequestSubmit : n/a -> n/a
-// Pull the new change request submitted to the form
+/**
+* Pulls the new change request submitted to the form
+*
+* @return {void}
```
Comment (jamescd18) [inline server-code/ChangeForm.js:9]:
```suggestion
```

File: server-code/ChangeForm.js [hunk 3]
```diff
@@ -3,8 +3,11 @@ Document: JS code connecting the Change Management Request form to the spreadshe
 
 */
 
-// onRequestSubmit : n/a -> n/a
-// Pull the new change request submitted to the form
+/**
+* Pulls the new change request submitted to the form
```
Comment (jamescd18) [inline server-code/ChangeForm.js:7]:
```suggestion
* Pulls the new change request submitted to the form, formats the Change Request data, and adds it to the database spreadsheet of all Change Requests. 
```

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Looking pretty good! Just a couple minor edits, and the fix looks pretty good to me.

**Review by jamescd18 — APPROVED** [review]
Positively lovely.

**Pull request merged**